### DISCUSSION TITLE
New Document from Template not working in SharePoint Online

### DISCUSSION THREAD

**Discussion by svanmoerkerke** [MicrosoftDocs/office-developer-client-docs#667] [Q&A]
I'm sorry to tell you but the New Document from Template example is not working.
The description mentioned: 'The following command will cause the application to create and open a new document based on the template stored at the specified URI. The template file is not modified in this process. An additional command argument may be supplied to specify the default path offered as a save location when the file is first saved. It is possible for the user to choose a different location.' is wrong!

When trying to use this syntax: "ms-word:nft|u|https://cohowinery/templates/elegance.pot|s|https://cohowinery/presentations" it will open Word with the template and the correct SharePoint metadata fields in the property pane, but the file is not automatically created as when a new file is created in the browser. When you close the file, Word will still ask you where to save the file, and will first suggest OneDrive. So you still have to go to the SharePoint library. So the save location option is doing nothing. Please fix this or modify the documentation accordingly.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: cc9d3afc-5f18-8c4f-1b9c-fd2fe52f49db
* Version Independent ID: a900d01a-0f42-13b8-3289-a2787f7b8499
* Content: [Office URI Schemes](https://docs.microsoft.com/en-us/office/client-developer/office-uri-schemes)
* Content Source: [docs/office-uri-schemes.md](https://github.com/MicrosoftDocs/office-developer-client-docs/blob/master/docs/office-uri-schemes.md)
* Product: **office365**
* GitHub Login: @o365devx
* Microsoft Alias: **o365devx**

**Comment by carlmarn** [comment]
I had the same issue and recently opened a support ticket, Case #:25734777, about this problem.
I previously have been using the Office URI scheme, to create a document from a document template, and the newly created document would be saved to a specific location.
The URI scheme that I used was:
ms-word:nft|u|[link-to-document-template]|s|[link-to-a-location-to-save-the-new-file]
The template and the save location are both in SharePoint online.
The result: Word creates a new document based on the template. SharePoint tries to read the SharePoint properties but seems to fail as no fields gets shown in the pane. When I save the document it proposes me to save the document to my OneDrive rather than the location that I specified in the URI.

**Comment by djjamba** [comment]
same here - edit and view URIs work but not nft
for good measure I tested under a site collection admin account - still no joy
I even placed the template (dotx) in the library i wanted to save to and it simply ignores the save |s| command
